packages/office-ui-fabric-react/src/components/Label/Label.scss

```diff
@@ -15,7 +15,7 @@
   color: $ms-color-neutralPrimary;
   box-sizing: border-box;
   display: block;
-  padding: 5px 0;
+  padding: 8px 0;
```

Comment:
@dzearing Should we remove the padding here and then deal with the regressions as they are discovered? Do we really want to comb through the entire project and look for Labels embedded in components and remove all their inherent padding and swap out for custom padding per example. How are we going to communicate to the Developers who use this framework that they need to implement their own 8px of padding when they use Label component? @betrue-final-final Ben? can we abandon this request? I wasn't expecting this level of push back on such a simple tweak...